### DISCUSSION TITLE
Volume notification issues

### DISCUSSION THREAD

**Discussion by borkog** [mikebrady/shairport-sync#1771] [Q&A]
Hi,
Don't know if it's a huge issue ... but it bugs me a little 

I use Hifiberry Beocreate 4ch amp with RPI4 and I'm trying to make my life easy. 

Whenever new song is played (normally or skip) newer iOS device sends volume info when the song begins. 

Why is that a problem for me? I change volume on my remote and when new songs comes along volume goes to one that is set by the iPhone.

If I play from the old iPad everything is nice (because old device do not send volume info when the song begins ... probably Airplay 1)

Is there a way to config ShairportSync to send volume info only when I really change the volume?

shairport-sync.conf file says:
`mixer_control_name = "DSPVolume";`

Thanks!

**Comment by mikebrady** [comment]
Thanks for the post. Not sure what can be done to help you in this situation. One slight possibility is to ignore the volume control information coming from the iOS device -- there's an `ignore_volume_control` setting. Would that work for you?
